src/System.Memory/ref/System.Memory.cs

```diff
@@ -91,13 +72,20 @@ public partial struct Span<T>
         public bool TryCopyTo(System.Span<T> destination) { throw null; }
     }
 
-    public static class SpanExtensions
+    public static class Span
```

Comment:
This assembly has never been shipped stable, as far as I know, so breaking changes are not a concern.